app/controllers/registrations_controller.rb

```diff
@@ -22,6 +22,8 @@ def create
     if recaptcha_disabled? || recaptcha_verified?
       build_resource(sign_up_params)
       resource.saw_onboarding = false
+      resource.registered = true
+      resource.registered_at = Time.current
```

Comment:
Do we need a backfill script?

Reply:
This is something that can be updated manually through admin, and I think it might be hard to identify which users would have gone through this path erroneously. It _may_ be technically possible to remedy this, but I think for this particular issue it's best to allow admins to deal with this manually if they recognize an issue.

Reply:
.........Actually scratch that. I think it does make sense. I realized it's logically easy enough to determine that if someone was not invited, they should by definition be registered. So I added a script. 👍